app/src/main/java/com/framgia/englishconversation/screen/timeline/TimelineViewModel.java

```diff
@@ -88,4 +91,11 @@ public TimelineAdapter getAdapter() {
     public void setAdapter(TimelineAdapter adapter) {
         mAdapter = adapter;
     }
+
+    @Override
+    public void onHeaderTouchListener(TimelineModel timelineModel) {
+        Bundle bundle = new Bundle();
+        bundle.putParcelable(Constant.ARGUMENT_TIMELINE, timelineModel);
+        mNavigator.startActivity(VideoDetailActivity.class, bundle);
```

Comment:
2.11 Argument in Activity and Fragment
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md